docs/getting-started/new-music-help-faq.md

```diff
@@ -0,0 +1,80 @@
+
+![Material Add Browse Mode](assets/new-music/material-add-browse.png){ align=left }
+
+## Frequently Asked Questions
+
+#### Why was the change made when New Music was working well for me?
+New Music browsing has actually had a long history of complaints and confusion going back 20 years because unless you carefully managed your file modified dates, simple things like updating tags could move and old album back to the top of the New Music list.  There are plenty of forum posts with questions, confusion and some crazy or clever workarounds.
+
+Over time most users fell into one of two groups. The first group learned to accept or even actually liked that updates to tags would cause an album to move back to the top of the list.  The second group were more New Music purists who would carefully manage their file modified dates by using tagging software that supported an option for preserving file modified dates. But as a result, this meant that a "Clear Library and Rescan All" would have to be run after each change to tags and required careful manual management of backing up those changes too. Now some folks got even more clever with the workarounds by using tools that would update the modified dates (after preserving them) to add just a few seconds so they can Scan for Changes and use automated tools for backup of those changes.
+
+The changes in 9.0 were to correct this long-standing problem by finally having a New Music list that was sorted by when Music was added to your Lyrion library while still maintaining a browse option for those who like a list of Recently Updated Albums sorted by file modified date. The great news here is this eliminates the need for the New Music purist to manage all those workarounds.  But if you prefer the workaround methods that you are used to, you can still continue to do so and use the new __Recently Updated Albums__ browse menu
+
+#### Why couldn't you just leave New Music as it has been for so long and create a new browse menu for the new thing?
+Change is hard when you have become accustomed to something, but can you imagine how confusing it would be to explain this to a new user who did not have your history. New Music needs to be the name for a list Music sorted by when it was added to the library and Recently Updated is the best name that was decided upon for the list sorted by file modified date.
+
+#### Why do I see New Album at the top of both New Music and Recently Updated Albums?
+Technically speaking neither of these queries use a means to know if an album existed in the library prior to the date and timestamp they sort by: 
+- __New Music__ is a list of your albums sorted on the date added field in your persist.db which represents when you added the music to your LMS library
```

Comment:
If you want to make this a bulleted list, you'll have to add empty lines between the lines, or they'll end up on one single line without bullets.